libqtile/core/manager.py

```diff
@@ -106,28 +107,23 @@ def __init__(
         self._stopped_event: Optional[asyncio.Event] = None
 
         self.server = IPCCommandServer(self)
-        self.load_config()
 
-    def load_config(self) -> None:
+    def load_config(self, initial=False) -> None:
         try:
             self.config.load()
             self.config.validate()
         except Exception as e:
             logger.exception('Error while reading config file (%s)', e)
-            self.config = confreader.Config()
-            from libqtile.widget import TextBox
-            widgets = self.config.screens[0].bottom.widgets  # type: ignore
-            widgets.insert(0, TextBox('Config Err!'))
+            on_fail = partial(send_popup, f"<b>Configuration error</b>\n{str(e)}")
+            send_notification("Configuration error", str(e), on_fail=on_fail)
```

Comment:
Should we use the same approach in `cmd_restart`? There's only a notification in that scenario with no fallback to a popup.

Reply:
Oh yeah, good point.

Reply:
Actually it's beginning to look like `send_popup` should just be part of `send_notification` as a general fallback in the case where a dbus notification server is not present. In which case, it might be worth adding a config option `disable_popups` for people like @tych0 who don't want any feedback if a config change triggers an error. Thoughts?

Reply:
Hmm. Yeah, I can see the reasoning for that. I would think that `disable_popups` (might need to change the name as some `Popup`s are still ok) should be `True` by default. If a user starts with an invalid config they'll get dropped into the default config and that way it's very obvious that something's wrong (i.e. no need for a popup). However, if they make changes to their config and try restarting, the popup would be helpful.

Reply:
How about `disable_notifications`, and it be used exclusively by `send_notification` (whether it uses `send_popup` or dbus)?

Reply:
Would that turn off notifications and popups? I think Tych0's objection was to the popups but a notification may still be acceptable.

Reply:
In that case it could be set to `False`, if the user wants them and has a notification server running.

Reply:
Ah, yes. Fair enough.